### PR TITLE
Work on viewing single post

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/client/components/Post.jsx [hunk 1]
```diff
@@ -1,35 +1,123 @@
...
+                                        <button className="commentReply" onClick={handleCommentReply(comment)}>Reply</button>
+                                        {userData.admin && (
+                                            <button className="commentDelete">Delete</button>
+                                        )}
+                                    </div>
+                                </div>
+                                {comment.children &&
+                                    comment.children.map((child, index) => {
+                                        return (
+                                            <div className="commentChild">
```
Comment (nathanSamosa) [inline src/client/components/Post.jsx:77]: love the rendering!

File: prisma/seed.js [hunk 2]
```diff
@@ -84,13 +84,56 @@ const createComment = async(user, post) => {
...
+                                id: comments[i].postId
+                            }
+                        },
+                        profile: {
+                            connect: {
+                                id: comments[i].profileId
+                            }
+                        }
+                    },
+                ]
```
Comment (TheunisK) [inline prisma/seed.js:133]: Well done for the work you did on this Lenni. Seamed very complicated to me.

File: src/client/components/Post.jsx [hunk 3]
```diff
@@ -1,35 +1,123 @@
...
+                {commentArr &&
+                    commentArr.map((comment, index) => {
+                        return (
+                            <div className="comment">
+                                <div className="individualComment">
+                                    <p className="commentUsername">{comment.profile.user.username}</p>
+                                    <p className="commentDateTime">{comment.createdAt}</p>
+                                    <p className="commentContent">{comment.content}</p>
+                                    <div className="commentInteract">
+                                        <button className="commentReply" onClick={handleCommentReply(comment)}>Reply</button>
```
Comment (TheunisK) [inline src/client/components/Post.jsx:68]: Is this the onClick that needs to change to a callback?

File: src/client/components/Post.jsx (hunk 1 above)
Comment (TheunisK) [reply src/client/components/Post.jsx:77]: I love everything Lenni does!

Comment (glowkeeper) [reply src/client/components/Post.jsx:77]: I'm loving the love :)

Comment (glowkeeper) [reply src/client/components/Post.jsx:77]: (and the conditional rendering ;) )

**Comment by glowkeeper** [comment]
Approved

**Pull request merged**